src/main.js

```diff
@@ -175,28 +168,30 @@ function makePoster(){
 }
 
 function saveMainPoster() {
-  currentPoster = new Poster(posterImg.src, posterTitle.innerHTML, posterQuote.innerHTML);
-    for(var i = 0; i < savedPosters.length; i++ ) {
-      if (savedPosters[i].url == currentPoster.url &&
-        savedPosters[i].title == currentPoster.title &&
-        savedPosters[i].quote == currentPoster.quote) {
-        var existingPoster = savedPosters[i]
-      }
-    }
-    if (!existingPoster) {
-      savedPosters.push(currentPoster);
-      makePoster();
-    }
+  if (!existingPoster()) {
+    savedPosters.push(currentPoster);
+    makePoster();
+  }
+}
+
+function existingPoster(){
+  return savedPosters.find((poster) => {
+    poster.imageURL === currentPoster.imageURL &&
+    poster.title === currentPoster.title &&
+    poster.quote === currentPoster.quote
+  })
 }
```

Comment:
better use of equals operator to check strictly equals. Nice.

Reply:
But I do wonder if this could be made easier. Do posters have ids? If not, could you create some at some point and then they would be usable here?

Reply:
Then you could just check the unique id instead of the url && title && quote. That's way too much work.  😂

Reply:
oh yeah I looked at your Poster class. You should def be using the id here.